src/components/utils/NotifCounter.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { createContext, useContext, useState, useEffect } from 'react'
+import { offchainWs } from './OffchainUtils'
+import { useMyAccount, checkIfLoggedIn } from './MyAccountContext';
+
+export type NotifCounterContextProps = {
+  unreadCount: number
+}
+
+export const NotifCounterContext = createContext<NotifCounterContextProps>({ unreadCount: 0 });
+
+export const NotifCounterProvider = (props: React.PropsWithChildren<{}>) => {
+  const { state: { address: myAddress } } = useMyAccount()
+
+  const [ contextValue, setContextValue ] = useState({ unreadCount: 0 })
+  const [ wsConnected, setWsConnected ] = useState(false)
+  const [ isLoggedIn, setIsLoggedIn ] = useState(false)
+  const [ address, setAddress ] = useState<string | undefined>()
+
+  if (checkIfLoggedIn() && !isLoggedIn) {

```

Comment:
Why do we need this?

Reply:
We should reconnect WS if account was changed.